lib/Controllers/list.js

```diff
@@ -95,8 +95,12 @@ List.prototype.fetch = function(req, res, context) {
     return result;
   }, {});
 
-  if (Object.keys(extraSearchCriteria).length)
-    criteria = _.assign(extraSearchCriteria, criteria);
+  if (Object.keys(extraSearchCriteria).length) {
+    if (Object.keys(criteria).length)
+      criteria = Sequelize.and(criteria, Sequelize.or.apply(null, [extraSearchCriteria]));
+    else
+      criteria = Sequelize.or.apply(null, [extraSearchCriteria]);
```

Comment:
@treefort make sure to ping me here when you get this done, github hasn't been emailing me for PR updates recently